interface.h

```diff
@@ -64,150 +67,853 @@
 // ----------------------------------------------------------------------------
 // timers
 
+/**
+ * @brief Adds a new timed event. An existing event timer at the provided index
+ *        identifier will be replaced.
+ *
+ * @param index Used to register and identify the timed event. Valid values are
+ *        between 0 and TIMED_EVENT_COUNT - 1
+ * @param ms How often the timed event should be triggered in milliseconds
+ * @param repeat 0 or 1 value indicating if the timed event should repeat
+ *        indefinitely
+ */
 void add_timed_event(uint8_t index, uint16_t ms, uint8_t repeat);
+
+/**
+ * @brief Stops and removes a previously added timed event at the index
+ *        identifier.
+ * 
+ * @param index Index identifier of the timed event to stop
+ */
 void stop_timed_event(uint8_t index);
+
+/**
+ * @brief Updates a previously registed timed event trigger interval in
+ *        milliseconds
+ * 
+ * @param index Index identifier of the timed event to update
+ * @param ms How often the timed event should be triggered in milliseconds
+ */
 void update_timer_interval(uint8_t index, uint16_t ms);
 
 
 // ----------------------------------------------------------------------------
 // clock
 
+/**
+ * @brief Get the global time value of milliseconds since start
+ * 
+ * @return uint64_t Returns milliseconds since start
+ */
 uint64_t get_global_time(void);
+
+/**
+ * @brief Check if hardware has a CV clock input
+ * 
+ * @return uint8_t Returns 0 if hardware does not have CV clock input
+ * @return uint8_t Returns 1 if hardware has CV clock input
+ */
 uint8_t has_clock_input(void);
+
+/**
+ * @brief Check if external CV clock input is connected
+ * 
+ * @return uint8_t Returns 0 if external CV clock is not connected
+ * @return uint8_t Returns 1 if external CV clock is connected
+ */
 uint8_t is_external_clock_connected(void);
+
+/**
+ * @brief Set the clock output on or off
+ * 
+ * @param on 0 if clock output should be off,
+ *           1 if clock output should be on
+ */
 void set_clock_output(uint8_t on);
 
 
 // ----------------------------------------------------------------------------
 // inputs
 
+/**
+ * @brief Get the number of hardware CV inputs available
+ * 
+ * @return uint8_t Returns number of hardware CV inputs
+ */
 uint8_t get_cv_input_count(void);
+
+/**
+ * @brief Get the CV value for the provided input index. Teletype is currently
+ *        the only Monome module that has a CV input.
+ * 
+ * @param index Hardware CV input number (first input is index 0)
+ * @return int16_t Returns CV value represented as a signed integer with the
+ *         range of -16,384 (-10v) and 16,383 (+10v).
```

Comment:
`range of -16,384 (-10v) and 16,383 (+10v)` -> `range of -16,384 (-10v) and MAX_LEVEL (+10v)`

this way it's another hint to use `MAX_LEVEL` if you need to scale voltage